### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 21

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#321] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152323529-192c392d-2007-4c25-8636-ab19713120cd.png)

**Comment by Exyss** [comment]
![Esercizio 21](https://user-images.githubusercontent.com/65043207/152661191-fbff6dfe-d455-4b6a-a356-6a7ea781b91d.jpg)

**Comment by Elia-Belli** [comment]
Piccolo errore nella dimostrazione di @Exyss, il caso base non è $n=0$ ma $n=1$, una sommatoria che va da 1 a 0 non avrebbe termini, tantomeno $A_0$.
- **Caso Base**: $n = 1$ , $A_1 = A_3 -1 \rightarrow 1 = A_1 + A_2 - 1 \rightarrow 1 = 2-1$
- **Ipotesi Induttiva**: $\forall n \geq 1, P(n)= \sum\limits_{i=1}^{n} A_i = A_{n+2}-1$
- **Passo Induttivo**: $n \geq 1$ , $\sum\limits_{i=1}^{n+1} A_i = A_{n+3}-1$
$\sum\limits_{i=1}^{n} A_i + A_{n+1} =A_{n+3} -1$ , applico $P(n)$
$A_{n+2}-1 + A_{n+1} = A_{n+3}-1$ , per definizione $A_{n+3} = A_{n+1} + A_{n+2}$, quindi il passo induttivo è dimostrato.
